app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,53 @@
+        Swipe down from the top of it to return to the home screen.
         \n
-        \nPress the button below the favorites panel to display the menu of the launcher.
+        \nThe button below the favorites panel opens the launcher menu.
         \n
-        \nIf you hold your finger a few moments on an application or a folder, you will be offered to access its settings.
+        \nLong-hold an app or folder to access its settings.
     </string>
     <string name="help_notification_title">Notification</string>
     <string name="help_notification_text">
-        The notification allows to display a popup with favorites above other applications when you leave the home screen.
-        In most cases, it should work directly and without issues.
+        Shows a popup of favorites above other apps when you leave the home screen.
         \n
-        \nHowever, in some special cases it may bring back to the home screen instead of being displayed over other applications.
-        \nIf you are facing this issue, there are two things you can try:
-        \n- Press the \"clear all\" button in Recent Apps
-        \n- Reboot your smartphone
-        \nAfter one of these actions, the problem should not appear again.
-        \nIf it persists, I am sorry but it seems this will not work for you :-/
+        \If instead it brings you back to the home screen:
+        \n- Press the \"clear all\" button in \"Recent Apps\" or
+        \n- Reboot your device
         \n
-        \nTo not be intrusive, the notification icon is transparent and low priority.
-        With some applications having even lower priorities, this may result in an empty space in the status bar.
+        \nUnintrusively, the notification icon is transparent and of low priority.
+        With some apps having even lower priorities, this may result in empty space in the status bar.
         \n
-        \nFinally, if you change the default launcher, you may still see the notification.
-        To hide it, you can open Discreet Launcher (no need to be the default launcher) and disable the notification in the settings.
+        \nIf the Discreet Launcher notification is still there after changing to a different launcher, you can
+        turn it off in the Discreet Launcher settings.
     </string>
     <string name="help_list_update_title">List update</string>
     <string name="help_list_update_text">
-        The list of applications and their icons are refreshed only after some events
-        (on start, when an application is added or removed, if the icon pack is modified).
-        When this happens, a small popup message appears during a few moments.
-        \n
-        \nIf something does not feel right, you should try to refresh the applications list manually.
+        Refreshing the app list and icons shows a small popup message, it also happens
+        when starting, when an app is added or removed, or if the icon pack is modified.

```

Comment:
Not very satisfied yet with this one, maybe also the part with the popup message should be a separate sentence.

The general idea of this paragraph is to say that there are some cases (ex: enable/disable apps, change language) where the list will not be automatically refreshed, it has to be done manually with the related menu button.
Since I don't have an exhaustive list of these cases, I indicated all cases for which it *will* be updated automatically and said that the manual refresh is for all other cases.